libraries/AP_GPS/AP_GPS_ERB.cpp

```diff
@@ -285,3 +287,4 @@ AP_GPS_ERB::_detect(struct ERB_detect_state &state, uint8_t data)
     }
     return false;
 }
+#endif
```

Comment:
```suggestion
#endif  // AP_GPS_ERB_ENABLED
```

similarly elsewhere

Reply:
on complete block removals the pattern has been NOT to add the comment to the endif....helpful for inline code for sure, however...

Reply:
@Hwurzburg just saving your round-trips here - I've seen tridge bounce PRs for not missing the endif on entire-block-removals.  Sometimes an `#endif` at the end of the file may *not* be a complete-block-removal - 'though no example springs to mind.